sympy/stats/crv.py

```diff
@@ -173,11 +176,26 @@ def check(*args):
 
     def sample(self, size=()):
         """ A random realization from the distribution """
-        icdf = self._inverse_cdf_expression()
         if not size:
-            return icdf(random.uniform(0, 1))
+            size = 1
+        if getattr(self,'_sample_python', None):
```

Comment:
Why not `hasattr`?

Reply:
Yes, `hasattr` would be good to use here. Thanks!